packages/create-turbo/src/index.ts

```diff
@@ -162,26 +162,32 @@ async function run() {
   );
 
   // merge package.jsons
-  let appPkg = require(path.join(sharedTemplate, "package.json"));
+  let sharedPkg = require(path.join(sharedTemplate, "package.json"));
+  let projectPkg = require(path.join(projectDir, "package.json"));
```

Comment:
These were renamed for readability. 

`sharedPkg` = the generic `pacakge.json` from `templates/_shared_ts/package.json`
`projectPkg` = the package manager specific `package.json` from `templates/{packageManager}/package.json`